.envrc

```diff
@@ -19,17 +19,29 @@ trap notice ERR
 # This is used to group issues on Sentry.io.
 # If an issue does not call info() or die() it will be grouped under this
 error_message="Unknown issue"
+# This has to be the same value as what sentry-cli accepts
 log_level="info"
 
-notice() {
-    [ $? -eq 0 ] && return
+help_message() {
+    cat <<EOF
+If you're stuck or have questions, ask in #discuss-dev-tooling.
+For direnv configuration info run: make direnv-help
```

Comment:
We could follow-up with this, but since we're condensing things, we could say something along the lines of: if you need more help run `make direnv-help`, and then have a more detailed help message. Additionally, `#discuss-dev-tooling` doesn't make sense if you're not a Sentry employee, so we could give a more specific help message in `make direnv-help`.

Reply:
I'm handling this in the next PR.